iopipe/contrib/trace/marker.py

```diff
@@ -41,12 +41,13 @@ def measure(self, name, start=None, end=None):
     def delete(self, name):
         self.timeline.delete(name)
 
-    def db_trace(self, trace, request):
+    def db_trace(self, trace, db_type, request):
         if self.context.instance.report is None:
             return
 
         entry = trace._asdict()
         entry["type"] = entry.pop("entryType")
+        entry["dbType"] = db_type
```

Comment:
Do we know this will be "Redis" at the time this is invoked?  In the JS modules, I've had to set the string in our code.

Reply:
Yes, it gets set by `collect_redis_metrics` in the `auto_db` module.